src/test/java/de/tum/in/www1/artemis/TutorLeaderboardServiceIntegrationTest.java

```diff
@@ -0,0 +1,89 @@
+package de.tum.in.www1.artemis;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.test.context.support.WithMockUser;
+
+import de.tum.in.www1.artemis.domain.Course;
+import de.tum.in.www1.artemis.domain.Exercise;
+import de.tum.in.www1.artemis.domain.enumeration.AssessmentType;
+import de.tum.in.www1.artemis.domain.modeling.ModelingExercise;
+import de.tum.in.www1.artemis.service.TutorLeaderboardService;
+import de.tum.in.www1.artemis.web.rest.dto.TutorLeaderboardDTO;
+
+public class TutorLeaderboardServiceIntegrationTest extends AbstractSpringIntegrationBambooBitbucketJiraTest {
+
+    @Autowired
+    private TutorLeaderboardService tutorLeaderboardService;
+
+    final private static int TUTOR_COUNT = 1;
+
+    final private static int ASSESSMENT_COUNT = 2;
+
+    Course course;
+
+    Exercise exercise;
+
+    /**
+     * Prepares the testing suite by initializing variables and mocks
+     */
+    @BeforeEach
+    public void reset() {
```

Comment:
I just added this suggestion to keep consistency between integration test classes
```suggestion
    public void initTestCase() {
```